### PR TITLE
Added markup

### PR DESCRIPTION
**Pull request by Alexander-Frost**
Marked up codebase so it is easier to navigate

### CODE REVIEW

**Comment by mredig** [comment]
Please use tabs for indentation - you can set your preference either in Xcode preferences or per file in a project. This tool can also help with doing so: https://apps.apple.com/us/app/tabifyindents-for-xcode/id1179234554?mt=12

File: Demo/CreateViewController.swift [hunk 1]
```diff
@@ -9,15 +9,20 @@
...
+    
 	@IBOutlet var titleTextField: UITextField!
 	@IBOutlet var subtitleTextField: UITextField!
 	@IBOutlet var widthTextField: UITextField!
 	@IBOutlet var heightTextField: UITextField!
 
-
-
+    // MARK: - Actions
+
```
Comment (mredig) [inline Demo/CreateViewController.swift:25]: Please eliminate trailing whitespace (the line is empty except for a whitespace character)

File: Demo/CreateViewController.swift [hunk 2]
```diff
@@ -9,15 +9,20 @@
...
+    // MARK: - Outlets
+    
 	@IBOutlet var titleTextField: UITextField!
 	@IBOutlet var subtitleTextField: UITextField!
 	@IBOutlet var widthTextField: UITextField!
 	@IBOutlet var heightTextField: UITextField!
 
-
-
+    // MARK: - Actions
```
Comment (mredig) [inline Demo/CreateViewController.swift:24]: Please leave no vertical space between a mark and the section it's leading.

File: Demo/DemoViewController.swift [hunk 3]
```diff
@@ -10,52 +10,60 @@ import UIKit
...
-			DispatchQueue.main.async {
-				if let error = error {
-					let alert = UIAlertController(error: error)
-					self?.present(alert, animated: true)
-				}
-				self?.tableView.refreshControl?.endRefreshing()
-				self?.tableView.reloadData()
-			}
-		}
-	}
```
Comment (mredig) [inline Demo/DemoViewController.swift:37]: It appears this section has been removed. Why?

File: Demo/DemoViewController.swift [hunk 4]
```diff
@@ -10,52 +10,60 @@ import UIKit
...
-				if let error = error {
-					NSLog("There was an error \(error)")
-				}
-				DispatchQueue.main.async {
-					self?.tableView.reloadData()
-					sender.isEnabled = true
-				}
-			})
-		}
-	}
```
Comment (mredig) [inline Demo/DemoViewController.swift:57]: It appears this section has been removed. Why?

File: Demo/DemoViewController.swift (hunk 3 above)
Comment (Alexander-Frost) [reply Demo/DemoViewController.swift:37]: It wasn't removed, just moved into the Actions tab

File: NetworkHandler/NetworkMockingSession.swift [hunk 5]
```diff
@@ -10,19 +10,25 @@ import Foundation
...
 		self.mockData = mockData
 		self.mockError = mockError
 		self.mockResponseCode = mockResponseCode
 		self.mockDelay = mockDelay
 	}
 
-	func loadData(with request: URLRequest, completion: @escaping (Data?, URLResponse?, Error?) -> Void) -> URLSessionDataTask? {
+    // MARK: - Public
+    
+	public func loadData(with request: URLRequest, completion: @escaping (Data?, URLResponse?, Error?) -> Void) -> URLSessionDataTask? {
```
Comment (mredig) [inline NetworkHandler/NetworkMockingSession.swift:31]: This is a good change, but it's best to keep each PR limited to a single theme of change. Adding MARK notation and editing function accessibility are two different categories.

Comment (Alexander-Frost) [reply NetworkHandler/NetworkMockingSession.swift:31]: No edits to code were made in this pr. Functions were only moved to make it easier to read.

File: Demo/DemoViewController.swift (hunk 4 above)
Comment (Alexander-Frost) [reply Demo/DemoViewController.swift:57]: Moved under Actions.

File: Demo/DemoViewController.swift (hunk 3 above)
Comment (mredig) [reply Demo/DemoViewController.swift:37]: Oh I see it now! My bad.

Comment (mredig) [reply Demo/DemoViewController.swift:37]: I figured it out - the indentation was changed from tabs to spaces, so git sees that as a complete removal, which is why this whole section is marked as removed (with the exact same code marked as inserted above). So, please fix this to use tabs for indentation.

File: NetworkHandler/NetworkMockingSession.swift (hunk 5 above)
Comment (mredig) [reply NetworkHandler/NetworkMockingSession.swift:31]: the `loadData` function had it's access modifier modified, as seen in the above diff.

**Pull request merged**